composer.json

```diff
@@ -24,6 +24,7 @@
         "doctrine/inflector": "~1.1"
     },
     "suggest": {
+        "doctrine/orm": "^2.5",
```

Comment:
In fact, the [suggests](https://getcomposer.org/doc/04-schema.md#suggest) section should take a description saying why the dependency can be useful, not a version bound ([conflict](https://getcomposer.org/doc/04-schema.md#conflict) should be used for that). We should correct it in another PR.